2015/07/17 17:41:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "isite/contact".' in Z:\domains\localhost\blogYii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('isite/contact')
#1 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blogYii/isite/contact
---
2015/07/17 18:14:24 [error] [php] session_regenerate_id(): Session object destruction failed (Z:\domains\localhost\blogYii\framework\web\CHttpSession.php:182)
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\auth\CWebUser.php(233): CWebUser->changeIdentity()
#1 Z:\domains\localhost\blogYii\framework\gii\models\LoginForm.php(43): CWebUser->login()
#2 Z:\domains\localhost\blogYii\framework\gii\controllers\DefaultController.php(43): LoginForm->login()
#3 Z:\domains\localhost\blogYii\framework\web\actions\CInlineAction.php(49): DefaultController->actionLogin()
#4 Z:\domains\localhost\blogYii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 Z:\domains\localhost\blogYii\framework\web\CController.php(286): DefaultController->runAction()
#6 Z:\domains\localhost\blogYii\framework\web\CController.php(265): DefaultController->runActionWithFilters()
#7 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(282): DefaultController->run()
#8 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 Z:\domains\localhost\blogYii\index.php(11): CWebApplication->run()
REQUEST_URI=/blogYii/index.php/gii/default/login
2015/07/17 19:07:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in Z:\domains\localhost\blogYii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('admin')
#1 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blogYii/index.php/admin
---
2015/07/18 18:22:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "page?view=about".' in Z:\domains\localhost\blogYii\framework\web\CController.php:483
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CController.php(270): CController->missingAction('page?view=about')
#1 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(282): CController->run('page?view=about')
#2 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page?view=...')
#3 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/blogYii/about
---
2015/07/18 18:23:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "page?view=about".' in Z:\domains\localhost\blogYii\framework\web\CController.php:483
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CController.php(270): CController->missingAction('page?view=about')
#1 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(282): CController->run('page?view=about')
#2 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page?view=...')
#3 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/blogYii/index.php/about
---
2015/07/18 18:24:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "page?view=about".' in Z:\domains\localhost\blogYii\framework\web\CController.php:483
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CController.php(270): CController->missingAction('page?view=about')
#1 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(282): CController->run('page?view=about')
#2 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page?view=...')
#3 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/blogYii/index.php/about
---
2015/07/18 18:25:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index.php/site/page?view=about".' in Z:\domains\localhost\blogYii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('index.php/site/...')
#1 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blogYii/about
---
2015/07/19 17:22:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "test".' in Z:\domains\localhost\blogYii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('test')
#1 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blogYii/test
---
2015/07/19 17:41:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Туцы".' in Z:\domains\localhost\blogYii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('\xD0\xA2\xD1\x83\xD1\x86\xD1\x8B')
#1 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blogYii/%D0%A2%D1%83%D1%86%D1%8B
---
2015/07/20 18:48:14 [error] [exception.CException] exception 'CException' with message 'Контроллер TestNewsController не может найти представление "data".' in Z:\domains\localhost\blogYii\framework\web\CController.php:878
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CController.php(782): CController->renderPartial('data', Array, true)
#1 Z:\domains\localhost\blogYii\protected\controllers\TestNewsController.php(11): CController->render('data', Array)
#2 Z:\domains\localhost\blogYii\framework\web\actions\CInlineAction.php(49): TestNewsController->actionData()
#3 Z:\domains\localhost\blogYii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\domains\localhost\blogYii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\domains\localhost\blogYii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(282): CController->run('data')
#7 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('testnews/data')
#8 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/blogYii/testnews/data
---
2015/07/20 18:53:14 [error] [exception.CException] exception 'CException' with message 'Контроллер TestNewsController не может найти представление "data".' in Z:\domains\localhost\blogYii\framework\web\CController.php:878
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CController.php(782): CController->renderPartial('data', Array, true)
#1 Z:\domains\localhost\blogYii\protected\controllers\TestNewsController.php(11): CController->render('data', Array)
#2 Z:\domains\localhost\blogYii\framework\web\actions\CInlineAction.php(49): TestNewsController->actionData()
#3 Z:\domains\localhost\blogYii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\domains\localhost\blogYii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\domains\localhost\blogYii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(282): CController->run('data')
#7 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('testnews/data')
#8 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/blogYii/testnews/data
---
2015/07/20 18:55:57 [error] [exception.CException] exception 'CException' with message 'Контроллер TestNewsController не может найти представление "data".' in Z:\domains\localhost\blogYii\framework\web\CController.php:878
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CController.php(782): CController->renderPartial('data', Array, true)
#1 Z:\domains\localhost\blogYii\protected\controllers\TestNewsController.php(11): CController->render('data', Array)
#2 Z:\domains\localhost\blogYii\framework\web\actions\CInlineAction.php(49): TestNewsController->actionData()
#3 Z:\domains\localhost\blogYii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 Z:\domains\localhost\blogYii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 Z:\domains\localhost\blogYii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(282): CController->run('data')
#7 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('testnews/data')
#8 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#9 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/blogYii/testnews/data
---
2015/07/21 16:45:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "rendertable".' in Z:\domains\localhost\blogYii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('rendertable')
#1 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blogYii/rendertable
---
2015/07/21 16:47:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "Rendertable".' in Z:\domains\localhost\blogYii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('Rendertable')
#1 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blogYii/Rendertable
---
2015/07/21 16:47:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "rendertable".' in Z:\domains\localhost\blogYii\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\domains\localhost\blogYii\framework\web\CWebApplication.php(141): CWebApplication->runController('rendertable')
#1 Z:\domains\localhost\blogYii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 Z:\domains\localhost\blogYii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blogYii/rendertable
---
